web/usuaris/src/__tests__/organisms/UsersTable.test.tsx

```diff
@@ -61,6 +64,7 @@ describe('UsersTable', () => {
           itinerarySlug: 'frontend-react',
           name: 'marc',
           dni: 'marc',
+          role: 'Administrador',
```

Comment:
To test the filter you have to test with the slug 'ADMIN' and add it to the handlers with the corresponding condition, am I right?